clone_install.sh

```diff
@@ -1,3 +1,4 @@
 git clone --shallow-since 2022-07-28 git@github.com:riveSunder/yuca.git
 cd yuca 
+git checkout b91ef7ab9cf468fda9a623fe88cac9c0623633bf
```

Comment:
✓improve repeatability by referencing specific commit for the yuca CA simulator.